editor/layout/style.scss

```diff
@@ -0,0 +1,7 @@
+.editor-layout.sidebar-opened .editor-layout__content {
+	margin-right: $sidebar-width;
+
+	.editor-text-editor__formatting {
```

Comment:
It needs to be nested under `.editor-layout.sidebar-opened`, I can rewrite it :)